docs/src/man/basics.md

```diff
@@ -0,0 +1,1220 @@
+  994 │   163     70  male        3  free     little           moderate        ⋯
+  995 │   186     74  female      3  free     little           moderate
+  996 │   430     74  male        1  own      little           NA
+  997 │   606     74  male        3  own      little           NA
+  998 │   756     74  male        0  own      little           rich            ⋯
+  999 │   330     75  male        3  free     little           little
+ 1000 │   536     75  female      3  own      NA               little
+                                                  4 columns and 985 rows omitted
+
+julia> german.Age[1] = 100
+100
+
+julia> german
+1000×10 DataFrame
+  Row │ id     Age    Sex     Job    Housing  Saving accounts  Checking accoun ⋯
+      │ Int64  Int64  String  Int64  String   String           String          ⋯
+──────┼─────────────────────────────────────────────────────────────────────────
+    1 │   391    100  female      1  rent     rich             moderate        ⋯
+    2 │   633     19  female      2  rent     little           NA
+    3 │    93     20  male        2  rent     NA               rich
+    4 │   155     20  female      2  rent     little           little
+    5 │   167     20  female      2  own      rich             moderate        ⋯
+    6 │   188     20  male        2  own      moderate         little
+    7 │   296     20  female      2  rent     NA               NA
+    8 │   410     20  female      2  own      little           moderate
+  ⋮   │   ⋮      ⋮      ⋮       ⋮       ⋮            ⋮                ⋮        ⋱
+  994 │   163     70  male        3  free     little           moderate        ⋯
+  995 │   186     74  female      3  free     little           moderate
+  996 │   430     74  male        1  own      little           NA
+  997 │   606     74  male        3  own      little           NA
+  998 │   756     74  male        0  own      little           rich            ⋯
+  999 │   330     75  male        3  free     little           little
+ 1000 │   536     75  female      3  own      NA               little
+                                                  4 columns and 985 rows omitted
+
+julia> german.Age
+1000-element Vector{Int64}:
+ 100
+  19
+  20
+  20
+  20
+  20
+  20
+  20
+  20
+  20
+   ⋮
+  68
+  68
+  70
+  74
+  74
+  74
+  74
+  75
+  75
+```
+
+Note, that in the above example the original `Age` vector is not mutated in the process.
+
+In-place functions are safe to call, except when a view of the `DataFrame` (created via a `view`, `@view` or `*groupby*`) or when a `DataFrame` created with `copycols=false` are in use.
+
+It is possible to have a direct access to a column `col` of a `DataFrame` `german` using the syntaxes `german.col`, `german[!, :col]`, via the `*eachcol*` function, by accessing a parent of a view of a column of a `DataFrame`, or simply by storing the reference to the column vector before the `DataFrame` was created with `copycols=false`.
```

Comment:
again - why is this comment relevant here?

Reply:
I had repeated 2nd paragraph again. Sorry.